Header header
uint8 num_detections
Detection[] detections
